docs/security/user-certs.md

```diff
@@ -126,43 +126,49 @@ If you have a CILogon issued certificate, contact [ca@cilogon.org](mailto:ca@cil
 Renewing your OSG user certificate
 ----------------------------------
 
-Your OSG user certificate can be renewed with either using the
+Your OSG user certificate can be renewed through the
 [OIM Web Interface](https://oim.opensciencegrid.org/oim/certificateuser)
-or the command line interface using
-[OSG PKI Tools](/security/certificate-management).
+or through the [command line interface](/security/certificate-management#osg-pki-command-line-clients):
+
+```
+user@host $ osg-user-cert-renew
+```
 
 ### Issues renewing your OSG user certificate
 
 One requirement for renewing your OSG user certificate is that your identity must be vetted within the last 5 years.
-You can see whether this requirement is met for your user certificate when *logged in* to the [OIM Web Interface].
+You can see whether this requirement is met for your user certificate when *logged in* to the
+[OIM Web Interface](https://oim.opensciencegrid.org/oim/certificateuser).
 
 !!! note
     When using the OIM Web Interface, make sure you are logged in and your name appears in the upper right corner.
     If it shows "Login" instead of your name, click it to log in.
 
 When you click on your user certificate, under the Renew tab, you will see a line item for
 "This certificate was approved within the last 5 years".
-If there is a red X on this line, then you will have to request a new certificate instead of renewing.
+If there is a red X on this line with no other red X's, then you will have to request a new certificate instead of
+renewing it.
 Your two options for requesting a new certificate are:
 
-1. [Request a certificate from CILogon](#getting-a-certificate-from-cilogon)
-1. If you need to keep your DN, or for whatever other reason you cannon use CILogon, you will need to:
-    1. Revoke your user certificate
-    2. Request a new user certificate with the same DN
-
-#### Revoking and re-requesting an OSG user certificate using the OIM Web Interface
-
-To revoke and re-request your OSG user certificate on the
-[OIM Web Interface](https://oim.opensciencegrid.org/oim/certificateuser)
-1. Login (your name should appear on the top right), and select your user certificate from the list.
-1. In a separate tab, open the [+Request New](https://oim.opensciencegrid.org/oim/certificaterequestuser) page.
-   We recommend opening the Request New page in a separate tab before revoking, in order to ensure that your CN gets
-   correctly populated in the Request New page.
-   Also, to ensure a prompt response, be sure that you select a Virtual Organization that has a Sponsor drop-down list,
```

Comment:
@brianhlin, just FYI, the language "to ensure a _prompt response_" was connected with the requirement for "a Virtual Organization _that has a Sponsor drop-down list_".  (It looks like you removed the language about the drop-down list.)

@timtheisen emphasised this point to me, that some VOs in the list have drop-down lists and some do not.  For instance, GLOW used to have one, but now does not, since the sponsor was Moate, and his cert recently expired, so now GLOW does not have a drop-down, thus @timtheisen chose the OSG VO and selected Mats from the list.

VOs without a drop-down list still allow specifying a sponsor manually, but the point of all this is, if you are not selecting a sponsor from a drop-down, your sponsor may not have a valid cert to be able to sponsor you, and thus you may _not_ get a prompt response, and you may be stuck for some time (perhaps longer than necessary) without a valid cert.

Reply:
I think that's addressed by saying "choose a VO and select a sponsor that will respond promptly"

Reply:
Ok, yeah my main thought was that it may not be clear to a user that if their normal sponsor (eg, Moate for GLOW) is not in the drop-down, then that sponsor may not be able to do their normal sponsor thing promptly.  But hopefully users will check with their sponsor first :)